frontend/app/controllers/Joiner.scala

```diff
@@ -293,15 +290,16 @@ object Joiner extends Controller with ActivityTracking with PaymentGatewayErrorH
         case t: Tier.Supporter if !upgrade => MembersDataAPI.Service.removeBehaviour(request.user)
         case _ =>
       }
-      Ok(views.html.joiner.thankyou(
+      Ok({salesforceService.metrics.putThankYou(tier)
```

Comment:
Have just realised that upgrades will also call this method (https://github.com/guardian/membership-frontend/blob/master/frontend/app/controllers/TierController.scala#L235), so in this form we are going to be counting new acquisitions and upgrades under a single metric.